data/styles/vehicle/include/Subways.mapcss

```diff
@@ -842,7 +842,7 @@ node|z17-[railway=station][transport=subway][city=tianjin] {icon-image: subway-t
 node|z16-[railway=subway_entrance][city=tianjin] {icon-image: subway-tianjin-s.svg;}
 
 /* Tokyo Subway Station */
-node|z13[railway=station][transport=subway][city=tokyo] {icon-image: subway-tokyo-s.svg;icon-min-distance: 1;}
+node|z13[railway=station][transport=subway][city=tokyo] {icon-image: none;}
```

Comment:
please add `text: none;` here also
default subway style in the vehicle mode has a label at z13 - we don't want to display a label without an icon

Reply:
Done. I will do some reading for my next change to styles to get to bottom of how everything works